actors/paych/tests/paych_actor_test.rs

```diff
@@ -454,6 +454,37 @@ mod update_channel_state_redeem {
         assert_eq!(sv.amount, ls_updated.redeemed);
         assert_eq!(sv.nonce, ls_updated.nonce);
     }
+
+    #[test]
+    fn redeem_voucher_nonce_reuse() {
+        let (mut rt, mut sv) = require_create_channel_with_lanes(3);
```

Comment:
original go test has 1 lane

Reply:
Well noted, @ZenGround0? Did you want to have a 1:1 parity here and switch this to 1 lane, or is 3 lanes giving us better coverage?